### DISCUSSION TITLE
any way to reset local database manually by deleting files on windows?

### DISCUSSION THREAD

**Discussion by kurakurakantor** [vrtmrz/obsidian-livesync#147] [Q&A] [answered]
i wonder where i could delete the local database manually, since uninstalling obsidian (and even deleting the appdata/roaming/obsidian) doesn't seem to do it.

i need this because i want to reconfigure all my sync, start from 0, but when i open obsidian (even after a fresh install) the livesync immediately restores the files i don't want

i know technically i could use the "reset local database" button, exit, then restore the vault like how i want it to, but i figure there might be a simpler way?

but i don't mind if that's the only option! thanks in advance!

**Comment by vrtmrz** [comment, marked as answer]
Thank you for asking me!

We can place `redflag.md` on the root of your vault. Self-hosted LiveSync will not perform anything if it exists.
And, one more feature.`Discard local database to reset or uninstall Self-hosted LiveSync` is under the Hatch pane.
They can be helpful for resurrecting from weird situations.

But uninstalling obsidian looks like doing so, what is your platform?

**Reply by kurakurakantor** [reply]
oh my gosh redflag.md is what i needed! i am on windows 11 by the way, somehow uninstalling it doesn't remove the obsidian data i'm not sure why, (on android uninstalling or removing data seems to do the trick). thank you for the fast reply!
